spec/prog/vm/nexus_spec.rb

```diff
@@ -476,7 +480,7 @@ def requested_disk_size(st)
     it "naps 12 second if it's the first time we execute wait_sshable" do
```

Comment:
we should probably also change the test description to "naps 10 ..."

Reply:
I didn't notice that. Thanks.